build.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id 'groovy'
+    id 'maven-publish'
+}
+
+group = 'io.jenkins.infra'
+version = '0.0.1'

```

Comment:
Our library is versioned. Check the tags.

Reply:
Changed version to the 1.20.2 (current version is 1.20.1, so I did a minor upgrade).

Reply:
hmm. You could just use the same version as the version indicates the API not the meta around it.

Reply:
Done.